docs/requirements.txt

```
@@ -3,4 +3,4 @@
 mkdocs==1.2.3
```

Mkdocs needs to be updated to be compatible (https://github.com/mkdocs/mkdocs/releases/tag/1.2.4):
```suggestion
mkdocs==1.2.4
```